trieste/models/model_interfaces.py

```diff
@@ -154,6 +155,7 @@ def __repr__(self) -> str:
 
     @property
     def model(self) -> SVGP:
+        """"""
```

Comment:
Do you not want the model property to appear in the docs?

Reply:
ah, ok, so this is a mistake. it should be on `__repr__`. will fix